third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -316,37 +382,24 @@ impl TestConfig {
         let mut ok = Self::check_diags(&mut test_output.borrow_mut(), &env);
 
         if ok {
-            trace!("After error check, GlobalEnv={}", env.dump_env());
-            // Flow-insensitive checks on AST
-            flow_insensitive_checkers::check_for_unused_vars_and_params(&mut env);
-            function_checker::check_for_function_typed_parameters(&mut env);
-            function_checker::check_access_and_use(&mut env, true);
-            ok = Self::check_diags(&mut test_output.borrow_mut(), &env);
-        }
-        if ok {
-            trace!(
-                "After flow-insensitive checks, GlobalEnv={}",
-                env.dump_env()
-            );
-            // Run inlining.
-            inliner::run_inlining(&mut env);
-            ok = Self::check_diags(&mut test_output.borrow_mut(), &env);
-        }
-        if ok {
-            trace!("After inlining, GlobalEnv={}", env.dump_env());
-        }
-
-        if ok {
-            function_checker::check_access_and_use(&mut env, false);
-            ok = Self::check_diags(&mut test_output.borrow_mut(), &env);
+            // Run env processor pipeline.
+            if self.dump_ast == AstDumpLevel::AllStages {
```

Comment:
We may eventually want to selectively dump stages, but I am ok implementing that when we have a concrete usecase.

Reply:
SG